src/lake/Lake/Build/Monad.lean

```diff
@@ -73,7 +73,15 @@ where
 
 /-- Busy wait to acquire the lock of `lockFile`, run `act`, and then release the lock. -/
 @[inline] def withLockFile [Monad m] [MonadFinally m] [MonadLiftT IO m] (lockFile : FilePath) (act : m α) : m α := do
-  try busyAcquireLockFile lockFile; act finally IO.FS.removeFile lockFile
+  try
+    busyAcquireLockFile lockFile; act
```

Comment:
```suggestion
    busyAcquireLockFile lockFile
    act
```
Since you're no longer trying to save on new lines. :-) Feel free to ignore of course.

Reply:
My hand was forced by the length of the line. 😆

Reply:
I am not opposed, but I do not think another CI build is worth it. ⏳